packages/shared-ui/src/components/Layout.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactNode } from 'react'
+
+interface LayoutProps {
+  children: ReactNode
+  header?: ReactNode
+  footer?: ReactNode
+  backgroundImageSrc?: string
+}
+
+export function Layout(props: LayoutProps) {
+  return (
+    <div
+      style={{ backgroundImage: props.backgroundImageSrc }}
+      className="relative flex min-h-screen flex-col overflow-hidden bg-repeat"
+    >
+      {props.header}
```

Comment:
Can we try just rendering the `Header` and `Footer` inline here? The `Moon` component is absolutely positioned anyway, so we might do something like:

```
<Layout>
  <Routes />
  <Moon />
</Layout>
```